curvefs/src/client/inode_wrapper.h

```diff
@@ -363,6 +377,19 @@ class InodeWrapper : public std::enable_shared_from_this<InodeWrapper> {
 
     CURVEFS_ERROR RefreshVolumeExtent();
 
+    bool NeedRefreshData() {
+        uint64_t size = GetS3ChunkSize();
+        if (size >= maxDataSize_ &&
+            ::curve::common::TimeUtility::GetTimeofDaySec()
+            - lastRefreshTime_ > refreshDataInterval_) {
```

Comment:
reverse these two conditions, `GetS3ChunkSize` is time-consuming.

Reply:
fixed